<launch>
  <!-- <include file="$(find ut_jackal)/graph_navigation/launch/velodyne.launch"/> -->
  <node pkg="ut_jackal" type="autonomy_arbiter" name="autonomy_arbiter"
      cwd="node" args="--config $(find ut_jackal)/config/autonomy_arbiter.lua" />
  <node pkg="ut_jackal" type="status_translator" name="status_translator"
      cwd="node" /> 
  <node pkg="ut_jackal" type="enml" name="enml" 
      args='-c="$(find ut_jackal)/config" -r "enml.lua"' 
      cwd="node" />
  <node pkg="graph_navigation" type="navigation" name="navigation" 
      args='-robot_config $(find graph_navigation)/config/navigation.lua velodyne_2dscan:=scan' 
      cwd="node"  />
  <node pkg="ut_jackal" type="websocket" name="websocket" cwd="node" >
    <remap from="scan" to="velodyne_2dscan_high_beams"/>
  </node>

  <include file="$(find ut_jackal)/launch/ouster.launch" />
  <include file="$(find zed_wrapper)/launch/zed2i.launch" />
</launch>


